gradle/sdk-registry.gradle

```diff
@@ -0,0 +1,83 @@
+apply plugin: 'maven-publish'
+apply plugin: 'com.mapbox.sdkRegistry'
+apply from: "${rootDir}/gradle/artifact-settings.gradle"
+
+version = project.ext.versionName
+group = project.ext.mapboxArtifactGroupId
+afterEvaluate {
+    publishing {
+        publications {
+            release(MavenPublication) {
+                from components.release
+                groupId this.group
+                artifactId project.ext.mapboxArtifactId
+                version this.version
+
+                artifact(androidSourcesJar)
+                artifact(androidJavadocsJar)
+
+                pom.withXml {
+                    final mainNode = asNode()
+                    mainNode.appendNode('name', project.ext.mapboxArtifactTitle)
+                    mainNode.appendNode('description', project.ext.mapboxArtifactTitle)
+                    mainNode.appendNode('url', project.ext.mapboxArtifactUrl)
+
+                    final licenseNode = mainNode.appendNode('licenses').appendNode('license')
+                    licenseNode.appendNode('name', project.ext.mapboxArtifactLicenseName)
+                    licenseNode.appendNode('url', project.ext.mapboxArtifactLicenseUrl)
+                    licenseNode.appendNode('distribution', "repo")
+
+                    final developerNode = mainNode.appendNode('developers').appendNode('developer')
+                    developerNode.appendNode('id', project.ext.mapboxDeveloperId)
+                    developerNode.appendNode('name', project.ext.mapboxDeveloperName)
+
+                    final scmNode = mainNode.appendNode("scm")
+                    scmNode.appendNode("connection", project.ext.mapboxArtifactScmUrl)
+                    scmNode.appendNode("developerConnection", project.ext.mapboxArtifactScmUrl)
+                    scmNode.appendNode("url", project.ext.mapboxArtifactUrl)
+                }
+            }
+        }
+    }
+}
+
+registry {
+    sdkName = project.ext.mapboxRegistrySdkName
+    override = project.ext.versionName.endsWith("-SNAPSHOT")
```

Comment:
I'd suggest we also include the commit hash in the snapshot version name, so we don't need to overwrite the same snapshot version again and again, user might not get new snapshots due to local cache.

Reply:
Sounds good, will align with the rest of maps snapshots names

Reply:
Added commit hash to the snapshot